Configuration/Settings.yaml

```diff
@@ -1,43 +1,35 @@
+
 MOC:
   Varnish:
-    # Disables Varnish caching if set to FALSE
-    enabled: TRUE
-    # URL or URLs (if array) Varnish is running on for purge requests (skip trailing slash)
+    enabled: true
     varnishUrl: 'http://127.0.0.1'
-    # Cache header sending configuration
     cacheHeaders:
-      # Default and maximum TTL in seconds
-      defaultSharedMaximumAge: ~
-      # Disable sending headers (useful for staging environments)
-      disabled: FALSE
-    # Port to use for reverse lookup in backend module
-    reverseLookupPort: ~
-    # List of ignored  cache tags to skip when flushing caches by tag, e.g. 'TYPO3.Neos:Document' which is used in 'TYPO3.Neos:Menu' elements
-    ignoredCacheTags: []
-    # Log configuration
```

Comment:
some comments are lost here due to the automated configuration migration, please adjust manually instead to and only change what's actually needed..